src/reducers/event-filters.js

```diff
@@ -1,9 +1,18 @@
 // @flow
-import type { Reducer } from "redux";
+import { DateTime } from "luxon";
 import type { EventFiltersAction } from "../actions/event-filters";
 import type { State } from "../data/event-filters";
+import type { NavigationAction } from "../actions/navigation";
+import type { InitAction } from "../actions";
+import { NAVIGATION } from "../actions/navigation";
+import { HOME, PARADE, SUPPORT_US, DONATE, SPONSOR } from "../constants/routes";
 
-const defaultState = {
+type Action = EventFiltersAction | NavigationAction | InitAction;
+
+const includedRoutes = [HOME, PARADE, DONATE, SPONSOR, SUPPORT_US];
```

Comment:
Should this be defined somewhere close to our navigation so we don't have to remember to update this if tabs change?

Reply:
Sure, I'll move it to the routes constants.